corehq/apps/app_manager/templates/app_manager/form_view_base.html

```diff
@@ -159,13 +159,13 @@
             {% if app.application_version == '2.0' %}
                 {% if allow_form_filtering %}
                 var putInRoot = {{ module.put_in_root|BOOL }};
-                var usercaseonlyModule = {{ module.is_usercaseonly|BOOL }}
+                var isUserCaseInUse = {{ is_usercase_in_use|BOOL }};
                 var formFilter = {{ form.form_filter|JSON }};
                 var allOtherFormsRequireACase = {{ form.all_other_forms_require_a_case|BOOL }};
                 var formFilterAllowed = ko.computed(function () {
                     return (allOtherFormsRequireACase &&
                             form_requires() === 'case' ||
-                            usercaseonlyModule) && !putInRoot;
+                            isUserCaseInUse) && !putInRoot;
```

Comment:
I think this expression needs an extra set of brackets: https://github.com/dimagi/commcare-hq/pull/7947/files#diff-c2c6b62a2f52e153a74b8b241d4938eeR166